flink-libraries/flink-table/src/main/scala/org/apache/flink/table/functions/utils/UserDefinedFunctionUtils.scala

```diff
@@ -268,23 +268,9 @@ object UserDefinedFunctionUtils {
   def getFieldInfo(inputType: TypeInformation[_])
```

Comment:
`getFieldInfo` is overridden in `StreamTableEnvironment`, so I cannot make it a static method.